testing/test_skipping.py

```diff
@@ -713,6 +714,27 @@ def doskip():
     assert result.ret == 0
 
 
+def test_skipped_folding(testdir):
+    testdir.makepyfile(
```

Comment:
Individual tests will have `pytestmark` in `keywords`. For example:
```
pytestmark = pytest.mark.skipif(False, reason="global skip")

@pytest.mark.skip(reason="individual mark")
def test_something():
    assert 3 == 3
```
will output:
```
SKIP [1] tests/test_skip.py:7: individual mark
```

Should I add test for any specific case?